app/models/app.js

```diff
@@ -17,6 +17,7 @@ export default DS.Model.extend(ProvisionableMixin, {
   lastDeployOperation: DS.belongsTo('operation', {async:true}),
   currentImage: DS.belongsTo('image', {async:true}),
 
+  reloadWhileProvisioning: true,
```

Comment:
I wonder if we should overload `shouldReload()` to only reload while in `deprovisioning` as well? Since an app is can be `pending` for a long time and we don't use `provisioning` for apps?